apps/web/test/lib/team-event-types.test.ts

```diff
@@ -13,18 +13,21 @@ it("can find lucky user with maximize availability", async () => {
     email: "test@example.com",
     bookings: [
       {
-        createdAt: new Date("2022-01-25"),
+        createdAt: new Date("2022-01-25T05:30:00.000Z"),
+      },
+      {
+        createdAt: new Date("2022-01-25T06:30:00.000Z"),
       },
     ],
   });
   const user2 = buildUser({
-    id: 1,
-    username: "test",
-    name: "Test User",
-    email: "test@example.com",
+    id: 2,
+    username: "test2",
+    name: "Test User 2",
+    email: "tes2t@example.com",
```

Comment:
Both users had same id, email, etc. Making this test more confusing than needed.